lua/lspinstall/servers/haskell.lua

```diff
@@ -0,0 +1,26 @@
+local config = require"lspinstall/util".extract_config("hls")
+config.default_config.cmd = { "./hls" }
```

Comment:
Just use `config.default_config.cmd = { "./haskell-language-server-wrapper", "--lsp" }` here instead of creating a separate executable.

Reply:
This doesn't work because `haskell-language-server-wrapper` looks for `haskell-language-server-XXX` in the PATH for right version. There are haskell language servers for ghc versions and `haskell-language-server-wrapper` is made to pick right one.

Reply:
Extracting `hls.tar.gz` in addition to the `haskell-language-server-wrapper`, extracts haskell-language-server-XXX files.

Reply:
I see, thanks :)